Lib/test/test_pathlib.py

```diff
@@ -1874,6 +1883,50 @@ def test_chmod(self):
         p.chmod(new_mode)
         self.assertEqual(p.stat().st_mode, new_mode)
 
+    @unittest.skipUnless(root_in_posix and len(all_users) > 1,
+                         "test needs root privilege and more than one user")
+    def test_chown_with_root(self):
+        # original uid and gid
```

Comment:
Please follow PEP 8.

```suggestion
        # The original uid and gid.
```